install/ecs/rds.tf

```diff
@@ -0,0 +1,57 @@
+
+# generate and store password in secret manager
+
+resource "random_password" "dekart_rds" {
+  length  = 16
+  special = false
+}
+
+resource "aws_secretsmanager_secret" "dekart_rds" {
+  name = "${var.dekart_deployment_name}-rds"
+}
+
+resource "aws_secretsmanager_secret_version" "dekart_rds" {
+  secret_id     = aws_secretsmanager_secret.dekart_rds.id
+  secret_string = random_password.dekart_rds.result
+  lifecycle {
+    ignore_changes = [
+      secret_string
+    ]
+  }
+}
+
+# subnet group
+
+resource "aws_db_subnet_group" "dekart_rds" {
+  name       = "${var.dekart_deployment_name}-rds"
+  subnet_ids = aws_subnet.private.*.id
+}
+
+# rds
+
+resource "aws_db_instance" "dekart" {
+  identifier                  = var.dekart_deployment_name
+  allocated_storage           = 10
```

Comment:
as I remember minimum allocated storage is 20gb

Reply:
Hard to say it's from example https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance

Reply:
You are actually right https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html